src/TinyGsmClientSIM7600.h

```diff
@@ -292,7 +533,21 @@ class TinyGsmSim7600 : public TinyGsmModem<TinyGsmSim7600>,
   /*
    * Secure socket layer (SSL) functions
    */
-  // No functions of this type supported
+ public:
+  bool addCertificate(const char* certificateName, const char* cert,
```

Comment:
Actually, you want to override `addCertificateImpl` and `deleteCertificateImpl` resp. here (and have it protected).